docs/docs/overview/quickstart/get-started-with-python.md

```diff
@@ -0,0 +1,198 @@
+| Chat    | Mattermost, Slack, Microsoft Teams, Discord                         |
+| Push    | FCM, APNS, Expo                                                     |
+
+Only one provider can be **active** per **channel**. Connect any of your favorite providers to get started. The email channel comes with Novu's email provider, which is active by default and includes 300 credits.
+
+## Create a Notification Workflow
+
+A notification workflow is the blueprint for the notifications that will be sent. It holds the entire flow of messages sent to the subscriber. This is where all the different channels are tied together under a single entity.
+
+The workflow includes the following:
+
+- Notification workflow name and Identifier
+- Channel tailored content:
+
+| Channel | Content Style                                                                                 |
+| ------- | --------------------------------------------------------------------------------------------- |
+| Email   | 1. Custom Code (HTML) with option to use custom variables via the handlebars , {{ }}, syntax. |
+|         | 2. Click and place UI items with the visual template editor.                                  |
+| SMS     | Text with the option to use handlebars syntax, {{ }} to inject custom variables.              |
+| Chat    | Text with the option to use handlebars syntax, {{ }} to inject custom variables.              |
+| In-App  | Text                                                                                          |
+
+Note: Proper authorization needs to be set for the Chat channel for subscribers.
+
+Please proceed to create a notification workflow.
+
+1. Click “Workflows” on the left sidebar of your Novu dashboard.
+2. Click the “Create Workflow” button on the top right.
+3. The name of a new workflow is currently "Untitled." Rename it to a more suitable title.
+4. Select "Email" as the channel you want to add.
+   ![set-email.png] (https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/set-email_wavtrn.png)
+
+5. Click on the recently added channel, fill the email subject and click “Update”.
+   ![update_email_template.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/update_email_template_ivn0jv.png)
+
+6. Click on the “Test” tab and send a test email to verify your notification workflow.
+   ![send_test_email.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/send_test_email_ngzmth.png)
+
+You should get an email within seconds. Yaaay, you have successfully sent your first notification via the Novu dashboard! Now, let’s take it a step further to trigger notifications via code.
+
+## Create A Subscriber
+
+The recipients of a triggered notification are called subscribers.
+
+Click “Subscribers” on the left sidebar of the Novu dashboard to see all subscribers. By default, the dashboard will display a subscriber, as you were added automatically during sign-up.
```

Comment:
Click “Subscribers” on the left sidebar of the Novu dashboard to see all subscribers. By default, the dashboard will display only one subscriber, as you were added automatically during sign-up.